src/main/java/core/basesyntax/interfaces/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.

Reply:
style check won't pass it